install_config/persistent_storage/topics/glusterfs_storageclass.adoc

```diff
@@ -41,6 +42,7 @@ Volume Options] for more GlusterFS volume options.
 <9> Optional. The
 link:https://docs.gluster.org/en/v3/Quick-Start-Guide/Architecture/[type of
 volume] to use.
+<10> Optional. Enables custom volume name support using the following format: <volumenameprefixstring>_<namespace>_<claimname>_UUID.
```

Comment:
Enclose the whole sample in ``.
Can we reasonably expect the user know what volumenameprefixstring and claimname are? If not, explain where to obtain them.